lib/moment/getMomentAdvancedInfo.ts

```diff
@@ -0,0 +1,43 @@
+import { Moment, MomentAdvancedInfo } from "@/types/moment";
+import selectSale from "@/lib/supabase/in_process_sales/selectSale";
+import getMomentOnChainInfo from "@/lib/viem/getMomentOnChainInfo";
+import { convertDatabaseSaleToApi } from "@/lib/sales/convertDatabaseSaleToApi";
+import { convertOnChainSaleToApi } from "@/lib/sales/convertOnChainSaleToApi";
+import selectMoments from "@/lib/supabase/in_process_moments/selectMoments";
+
+export const getMomentAdvancedInfo = async (moment: Moment): Promise<MomentAdvancedInfo> => {
```

Comment:
This is getting the Moment `sale` info right?
- actual: getMomentAdvancedInfo
- suggested: getMomentSale